.github/workflows/bump-version-release.yaml

```diff
@@ -57,34 +52,35 @@ jobs:
         uses: bakdata/ci-templates/actions/checkout@1.32.0
         with:
           ref: ${{ github.event.repository.default_branch }}
-          persist-credentials: false # required for pushing to protected branch later
+          persist-credentials: true # required for pushing to protected branch later
 
-      - name: Bump version
+      - name: Setup Git
+        run: |
+          git config --global user.name "${{ secrets.github-username }}"
+          git config --global user.email "${{ secrets.github-email }}"
+
+      - name: Remove snapshot suffix
         id: bump-version
-        uses: bakdata/ci-templates/actions/bump-version@v1.21.2
+        uses: bakdata/ci-templates/actions/bump-version@feat/new-bumpversion
         with:
-          release-type: ${{ inputs.release-type }}
+          release-type: "release"
           working-directory: ${{ inputs.working-directory }}
 
+      - name: Commit .bumpversion.cfg file without snapshot
+        run: |
+          git add .bumpversion.cfg
+          git commit -m "Bump version ${{ steps.bump-version.outputs.old-version }} → ${{ steps.bump-version.outputs.release-version }}"
+          git fetch origin
+          git rebase --strategy-option=theirs origin/main
+          git push --verbose
+
       - name: Create changelog
         id: build-changelog
-        uses: bakdata/ci-templates/actions/changelog-generate@1.35.0
+        uses: bakdata/ci-templates/actions/changelog-generate@tiedemann/changelog-action
```

Comment:
Did you test the case where we also want to have the up to date snapshot in a file on the default branch and the tag?